src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -1416,6 +1416,38 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         <Extension>
+        Public Function GetInConversion(compoundOperation As ICompoundAssignmentOperation) As Conversion
```

Comment:
>GetInConversion [](start = 24, length = 15)

Are all code paths in this function covered by unit-tests?

Reply:
They are now.